src/components/UserActions/UserActions.ts

```diff
@@ -318,16 +317,33 @@ export class UserActions extends Component {
     private renderNoActions() {
         const element = document.createElement('div');
         element.classList.add('coveo-no-actions');
-        element.innerText = l(`${UserActions.ID}_no_actions`);
+        element.innerHTML = `
+        <div class="coveo-user-actions-title">${l(UserActions.ID)}</div>
+        <div>${l(UserActions.ID + '_no_actions_title')}.</div>
+        <br/>
```

Comment:
Instead of adding `<br />` tags like this, you could add a class like `coveo-no-actions-section` to add the top margin you want.

Reply:
I wrote a comment but it disappeared :( 

Basically the thing we are displaying I consider more as a 1 paragraph than multiple HTML elements. What do you think? It felt more like a line break, than a margin between elements.